bitnami/spark/values-production.yaml

```diff
@@ -259,6 +259,69 @@ worker:
     ## Max number of workers when using autoscaling
     replicasMax: 10
 
+## Metrics configuration
+##
+metrics:
+  enabled: enabled
+
+  ## Prometheus metrics service parameters
+  ##
+  service:
+    ## Metrics port
+    ##
+    port: 9117
+    ## Annotations for the Prometheus metics service
+    ##
+    annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "{{ .Values.metrics.service.port }}"
+
+  ## Prometheus Service Monitor
+  ## ref: https://github.com/coreos/prometheus-operator
+  ##      https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+  ##
+  serviceMonitor:
```

Comment:
We are saying `serviceMonitor` but we are using `PodMonitor` instead, could we align this?

Reply:
Sure, :facepalm: